sale_order_alternate_name/__manifest__.py

```diff
@@ -0,0 +1,23 @@
+# Copyright 2021 NuoBiT Solutions - Kilian Niubo <kniubo@nuobit.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl)
+
+{
+    "name": "Sale order alternate name",
+    "version": "12.0.1.0.0",
+    "author": "NuoBiT Solutions, S.L., Kilian Niubo",
+    "license": "AGPL-3",
+    "category": "Sale",
+    "website": "https://github.com/OCA/pms",
+    "summary": "This module deletes all the tasks linked to a sale order"
```

Comment:
This is not what the module does, please fix it. You can use the same description as Readme.rst